demos/noise_suppression_demo/python/noise_suppression_demo.py

```diff
@@ -52,22 +52,22 @@ def wav_read(wav_name):
     with wave.open(wav_name, "rb") as wav:
         if wav.getsampwidth() != 2:
             raise RuntimeError("wav file {} does not have int16 format".format(wav_name))
-        if wav.getframerate() != 16000:
-            raise RuntimeError("wav file {} does not have 16kHz sampling rate".format(wav_name))
```

Comment:
Please, remove 16kHz mentions from arg help. Same for c++

Reply:
@Wovchena why ? if it is known limitation, why not mention that?

Reply:
The limintation is being removed because
```
       if wav.getframerate() != 16000:
            raise RuntimeError("wav file {} does not have 16kHz sampling rate".format(wav_name))
```
is deleted

Reply:
ok. to avoid confusion I will back this check in other way.